packages/core/src/common/command.ts

```diff
@@ -40,6 +40,9 @@ export interface Command {
      * A category of this command.
      */
     category?: string;
+
+    // tslint:disable-next-line:no-any
+    props?: { [key: string]: any }
```

Comment:
Why do we need this one? It would be good to keep scope of changes to git/scm extensions and don't touch other extensions. The PR is big already enough.

Reply:
I need it to pass additional arguments e.g. group

Reply:
please revert it and downcast in scm extension